tools/xcodeproj_shims/installers/lldb-settings.sh

```diff
@@ -20,3 +20,10 @@ settings append target.source-map ./ "$BAZEL_WORKSPACE_ROOT/"
 settings set target.sdk-path $SDKROOT
 settings set target.swift-framework-search-paths $FRAMEWORK_SEARCH_PATHS
 END
+
+if [ -v BAZEL_LLDB_SWIFT_EXTRA_CLANG_FLAGS ]
+then
+cat <<-END > ~/.lldbinit-source-map
```

Comment:
I wonder since we are modifying this file `~/.lldbinit-source-map` why not modify `~/.lldbinit` as well here? Is it because this is our own file while .lldbinit can be shared by multiple projects/other IDE that uses lldb?

Reply:
I think it's because the developer owns ~/.lldbinit and should be allowed to put what they want in there without us clobbering it. For example, they might put Tulsi LLDB debug settings in there too or any custom debug settings they want xcode to pick up.

If we ask the developer to explicitly include this other file (~/lldbinit-source-map) in their ~/lldb-init, then they maintain control of the contents of lldbinit.